specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -492,6 +635,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all government-issued active storms.",
+        "operationId": "Weather_GetActiveStorms",
+        "x-ms-examples": {
+          "GetActiveStorms": {
+            "$ref": "./examples/GetActiveStorms.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ActiveStormResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/{format}": {
+      "get": {
+        "description": "**Get government-issued Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nSearch government-issued storms by year, basin ID, and government ID.",
+        "operationId": "Weather_StormSearch",
+        "x-ms-examples": {
+          "StormSearch": {
```

Comment:
You may also consider friendly names for those example: like https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Render/preview/2.1/render.json#L61